tests/test_all_fields.py

```diff
@@ -37,33 +37,40 @@
     'invoices': set()
 }
 
+# `updated_by_event_type` field's value available in the records only if records are emitted by `event_updates`.
 FIELDS_TO_NOT_CHECK = {
     'customers': {
         # Below fields are deprecated or renamed.(https://stripe.com/docs/upgrades#2019-10-17, https://stripe.com/docs/upgrades#2019-12-03)
         'account_balance',
         'tax_info',
         'tax_info_verification',
         'cards',
-        'default_card'
+        'default_card',
+        'updated_by_event_type'
```

Comment:
Can we add `updated_by_event_type` to `FIELDS_ADDED_BY_TAP` map instead of `FIELDS_TO_NOT_CHECK ` because its look like an above field is added by tap as same as `updated` field?

Reply:
`updated_by_event_type` field's value will be populated on all records emitted by `event_updates` only. Here in this test case, we are performing assertion on records emitted by the actual stream endpoint only. So, the `updated_by_event_type` field's value will not be available in such records. That's why we skipped this field.